src/app-layout/visual-refresh-toolbar/toolbar/trigger-button/styles.scss

```diff
@@ -63,13 +62,7 @@
   }
 }
 
-/*
-Warning! If these design tokens for width change it will adversely impact 
-the calculation used to determine the Split Panel maximum width in the 
-handleSplitPanelMaxWidth function in the context.
-*/
```

Comment:
This comment is copy-paste from VR app layout to the toolbar version, but it is not relevant anymore, because the implementation changed completely